packages/e2e-tests/specs/editor/blocks/navigation.test.js

```diff
@@ -468,9 +468,7 @@ describe( 'Navigation', () => {
 		);
 		const createPagePromise = createPageButton.click();
 		await Promise.all( [ responsePromise, createPagePromise ] );
-		expect( console ).toHaveErroredWith(
-			'Failed to load resource: the server responded with a status of 404 (Not Found)'
-		);
+		expect( console ).toHaveErrored();
```

Comment:
Or if we want to be extra clear about the error we're expecting we could do a substring match. Probably not worth it though. `toHaveErrored` is probably good enough.

```suggestion
		expect( console ).toHaveErroredWith(
		expect.stringContaining( 'Failed to load resource'
		));
```